features/support/ui/pages/money_manager.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../page'
+
+module UI::Pages
+  class MoneyManager < UI::Page
+    set_url '/en/tools/money-manager/'
+
+    element :received_first_payment_true,
+            '#questionnaire_received_first_payment_true'
```

Comment:
In order to de-couple test from actual CSS implementations we tend to add descriptive classes to the field and then reference it in the test. 

Within the view file we add this class `.t-questionnaire_received_first_payment_true` to the field. And then set it as the reference in the page objects as follows, `element :received_first_payment_true, '.t-questionnaire_received_first_payment_true'